Manifests/ManagedPreferencesApplications/com.microsoft.Edge.plist

```diff
@@ -2159,6 +2162,38 @@ By default, all extensions are whitelisted, but if all extensions have been blac
 			<key>pfm_type</key>
 			<string>array</string>
 		</dict>
+		<dict>
```

Comment:
Can you sort the `dict` for `PolicyRefreshRate` alphabetically like the other keys please.